### PR TITLE
fix importing of avatar entities

### PR DESCRIPTION
**Pull request by sethalves**
- fix importing of avatar entities, esp those which are a child of the avatar

### CODE REVIEW

**Comment by sethalves** [comment]
to test:
- save this json file to your local disk
```javascript
{
    "Entities": [
        {
            "clientOnly": 1,
            "collidesWith": "",
            "collisionMask": 0,
            "collisionless": 1,
            "created": "2017-10-02T23:02:57Z",
            "dimensions": {
                "x": 0.5,
                "y": 0.5,
                "z": 0.5
            },
            "id": "{e2349542-bcdc-4578-a583-2f083bf045d5}",
            "ignoreForCollisions": 1,
            "lastEdited": 1506985436198104,
            "lastEditedBy": "{d522ba6c-e2cb-453a-8361-4688f478044a}",
            "modelURL": "http://headache.hungry.com/~seth/hifi/sphere/sphere.obj",
            "name": "XXX XXX",
            "owningAvatarID": "{00000000-0000-0000-0000-000000000000}",
            "parentID": "{00000000-0000-0000-0000-000000000001}",
            "parentJointIndex": 63,
            "queryAACube": {
                "scale": 2.598076105117798,
                "x": -1.299038052558899,
                "y": -1.299038052558899,
                "z": -1.299038052558899
            },
            "rotation": {
                "w": -0.1340387463569641,
                "x": -0.015055418014526367,
                "y": 0.9847862720489502,
                "z": -0.10955753922462463
            },
            "shapeType": "sphere",
            "type": "Model",
            "userData": "{\"grabbableKey\":{\"grabbable\":1}}"
        }
    ],
    "Version": 73
}
```
- use CREATE to import it
- see that there is now a mirrored sphere on your head
- more soon

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3921/HighFidelity-Beta-PR11532-b96fac2f0064062902f060d025585c30f4e09e1a.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3921/HighFidelity-Beta-PR11532-b96fac2f0064062902f060d025585c30f4e09e1a.exe

File: libraries/entities/src/EntityTree.cpp [hunk 1]
```diff
@@ -1935,6 +1941,12 @@ bool EntityTree::readFromMap(QVariantMap& map) {
             entityItemID = EntityItemID(QUuid::createUuid());
         }
 
+        auto nodeList = DependencyManager::get<NodeList>();
+        const QUuid myNodeID = nodeList->getSessionUUID();
+        if (properties.getClientOnly()) {
+            properties.setOwningAvatarID(myNodeID);
+        }
+
```
Comment (AndrewMeadows) [inline libraries/entities/src/EntityTree.cpp]: `myNodeID` is only used inside the `if` context, but is not necessary for entering that context --> we should move the myNodeID initialization into the `if` context.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3926/HighFidelity-Beta-PR11532-30269c6794aaaed88a298b4c5c05f020f8ded87d.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3926/HighFidelity-Beta-PR11532-30269c6794aaaed88a298b4c5c05f020f8ded87d.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3942/HighFidelity-Beta-PR11532-8d0073162081151d6f8d51aaca858336c3133383.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3942/HighFidelity-Beta-PR11532-8d0073162081151d6f8d51aaca858336c3133383.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3946/HighFidelity-Beta-PR11532-195d96bd43cb86a8e82ce91890f55ec32608af51.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3946/HighFidelity-Beta-PR11532-195d96bd43cb86a8e82ce91890f55ec32608af51.exe

**Comment by YuppyPlays** [comment]
Testing.

**Comment by YuppyPlays** [comment]
Testing complete:

save this json file to your local disk **Check**
use CREATE to import it **Check**
see that there is now a mirrored sphere on your head **Check**
more soon **Will wait for more**

**Pass**

**Pull request merged**